overtime/algorithms/sliding_window_temporal_vertex_cover.py

```diff
@@ -0,0 +1,244 @@
+import overtime as ot

```

Comment:
please modify the comments in the files according to the coding style in other files. See e.g. https://github.com/overtime3/overtime/blob/dev/overtime/algorithms/foremost.py